docroot/themes/custom/govcms_uikit_starterkit/sass/layout/_views.scss

```diff
@@ -4,6 +4,10 @@
 //
 
 .view {
+  .views-infinite-scroll-content-wrapper {
+    display: flex;
```

Comment:
All contact cards are currently displayed inline. Adding `flex-wrap: wrap;` should fix this.